appendix/device-checklist.md

```diff
@@ -10,33 +10,35 @@ Before starting, gather a list of all devices to audit that are used for Lullabo
 
 With each device, validate:
 
-- [ ] All computers have a complex password with a screen lock timeout set to no more than 10 minutes.
-- [ ] All computer hard drives are encrypted with FileVault, Bitlocker, or equivalent.
-- [ ] All operating systems and browsers should be patched and up to date.
-- [ ] Confirm VPN app (Proton VPN) is setup and the team member is aware of the "split tunnel" option.
-- [ ] Confirm 1Password is installed with browser extensions.
-- [ ] Confirm an adblocker (uBlock origin, AdGuard) is installed in the default browser.
+- [ ] Confirm all mobile devices have a [screen timeout set to no more than 10 minutes](https://security.lullabot.com/physical/locking.html).
+- [ ] Confirm "Find My" or equivalent is enabled.
+- [ ] All operating systems should be up to date.
+- [ ] All browsers should be up to date.
```

Comment:
I don't think this is in our list, but I'm not against adding it. And I'm not sure "all" makes sense, especially where folks may have little used browsers that are behind (until they open it for this check!).

Reply:
Yeah, browsers up to date was on this list, but not our spreadsheet.
I like it on the self checklist.

I agree, I'm unlikely to normally update a browser until I use it.